### PR TITLE
retrieval: double timeout in target scrape test.

### PR DESCRIPTION
**Pull request by fabxc**

### CODE REVIEW

File: retrieval/target_test.go [hunk 1]
```diff
@@ -293,7 +293,7 @@ func TestTargetScrapeTimeout(t *testing.T) {
 	)
 	defer server.Close()
 
-	testTarget := newTestTarget(server.URL, 25*time.Millisecond, clientmodel.LabelSet{})
```
Comment (juliusv) [inline retrieval/target_test.go]: Shouldn't this stay at 25, but we just sleep longer to have a higher chance of hitting the deadline?

Comment (fabxc) [inline retrieval/target_test.go]: The reported issue was that a request timed out that shouldn't have. I.e. when execution reached the request the 25ms have already passed. So we don't want to hit the deadline.

For the case where we want to hit the timeout even the 5ms are unnecessary.

Comment (juliusv) [inline retrieval/target_test.go]: Ah, ok.

**Comment by juliusv** [comment]
:+1:

**Pull request merged**